### PR TITLE
Add Utility class to create menu links. See BASIC-1918

### PR DESCRIPTION
**Pull request by jbickar**
**READY FOR REVIEW**

### CODE REVIEW

File: Stanford/Utility/Install/CreateMenuLinks.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+/**
+ * @file
+ * This function creates a new Menu Position rule.
+ */
+
+namespace Stanford\Utility\Install;
+
+/**
+ * Create new menu links.
+ * @param array $mp_rules A multidimensional array with the following keys:
```
Comment (sherakama) [inline Stanford/Utility/Install/CreateMenuLinks.php]: Move this doc block to the execute method.
Also, please update the variable names to match.

Comment (jbickar) [inline Stanford/Utility/Install/CreateMenuLinks.php]: WTF copypasta?

File: Stanford/Utility/Install/CreateMenuLinks.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ */
+
+namespace Stanford\Utility\Install;
+
+class CreateMenuLinks extends \AbstractInstallTask {
+
+
+  public function execute(&$args = array()) {
+    /**
+     * Create new menu links.
```
Comment (sherakama) [inline Stanford/Utility/Install/CreateMenuLinks.php]: Please move doc block above function.

Comment (jbickar) [inline Stanford/Utility/Install/CreateMenuLinks.php]: I was wondering why you said to move it to the execute function. And why I blindly obeyed.

Comment (sherakama) [inline Stanford/Utility/Install/CreateMenuLinks.php]: Yeah sorry, I meant, above function and below class

**Pull request merged**